src/hooks/useBankTransactions/useBankTransactions.tsx

```diff
@@ -414,6 +414,7 @@ export const useBankTransactions: UseBankTransactions = params => {
     if (refreshTrigger !== -1) {
       refetch()
       refetchAccounts()
+      eventCallbacks?.onTransactionsFetched?.()
```

Comment:
I am on board with the principle of erring on the side of over communicating, but this feels a LITTLE extreme since we do this with polling a lot. 

I'd propose we at least move this down to line ~441 where we only would trigger the callback if the data actually changes.

Reply:
Makes sense- done.